src/actions.test.js

```diff
@@ -83,4 +85,36 @@ describe('actions', () => {
       });
     });
   });
+
+  describe('loadRestaurant', () => {
+    context('with selectedRestaurant', () => {
+      beforeEach(() => {
+        store = mockStore({
+          selectedRestaurant: 1,
+        });
+      });
+
+      it('runs setRestaurant', async () => {
+        await store.dispatch(loadRestaurant());
+
+        const actions = store.getActions();
+
+        expect(actions[0]).toEqual(setRestaurant([]));
```

Comment:
```suggestion
        expect(actions[0]).toEqual({ type: 'setRestaurant', payload: { restaurant } });
```
요렇게 명시적으로 실제 액션 객체를 표현해주면 무슨일이 벌어지는지 읽는 입장에서 파악하기 쉽죠.